src/components/common/NewActionButton/NewActionButton.tsx

```diff
@@ -59,7 +59,7 @@ const MSG = defineMessages({
 //   };
 // }
 
-const ColonyHomeActions = (/* { colony, ethDomainId}: Props */) => {
+const NewActionButton = ({ colony, ethDomainId }: Props) => {
```

Comment:
colony / ethdomain are not being used, please either comment them out or make sure they are used